dist/profile/manifests/buildmaster.pp

```diff
@@ -211,7 +211,7 @@
   ##############################################################################
 
   docker::run { 'jenkins':
-    image            => 'jenkins/jenkins:lts-alpine',
+    image            => 'jenkins/jenkins:lts-jdk11',
```

Comment:
This will configure every jenkins instance, including trusted and cert. 
Instead we must parametrize this variable and specifically configure ci.jenkins.io client with that tag

Reply:
This [PR](https://github.com/jenkins-infra/jenkins-infra/pull/1305) parametrize this puppet class

Reply:
This is definitively not what I wanted. Will I still be able to change the `JAVA_OPTS` then?

Reply:
Variables defined [here](https://github.com/jenkins-infra/jenkins-infra/blob/staging/dist/profile/manifests/buildmaster.pp#L33-L17), can be over-ridden per client in the hieradata directory, ci.j.io client configuration is located [here](https://github.com/jenkins-infra/jenkins-infra/blob/staging/hieradata/clients/ci.yaml)  
Now that I parametrized the docker image, you can just add a new variable in the client hieradata.
We must also parametrized JAVA_OPTS in order to use different options per environment